.github/workflows/ci.yml

```diff
@@ -600,3 +600,200 @@ jobs:
       - name: jscheck
         run: |
           make jscheck
+
+  meson:
+    runs-on: ${{ matrix.platform }}
+    name: meson on ${{ matrix.platform }} (${{ matrix.mode.name }} ${{ matrix.flavor }}, ${{ matrix.features.name }})
+    strategy:
+      fail-fast: false
+      matrix:
+        flavor:
+          - debug
+          - release
+        features:
+          - name: default
+            args: ""
+          - name: libc
+            args: -Dlibc=true
+          - name: mimalloc
+            args: -Dcli_mimalloc=auto
+          - name: jscheck
```

Comment:
I don't think this works, can you please remove it? It's not even part of the CMake build.

Reply:
oooh i think i accidently forgot some leftovers from testing. cleaning it.